HelperFunctions/ThirdParty/questdlg_nonmodal.m

```diff
@@ -0,0 +1,602 @@
+% Lines in comments and 418 only lines modified from MATLAB's 2019b questdlg
```

Comment:
If it were possible, I would have called `questdlg` in a wrapper which modified this seemingly tiny figure property. Given that the uiwait was called inside questdlg, there is no such opportunity to do so :(